zcash_client_sqlite/src/wallet.rs

```diff
@@ -3194,6 +3194,7 @@ pub mod testing {
 
         let results = stmt
             .query_and_then::<TransactionSummary<AccountId>, SqliteClientError, _, _>([], |row| {
+                let raw_tx: Vec<u8> = row.get("raw")?;
```

Comment:
This will break (either loudly, or silently with an empty `Vec` which is a bad failure mode), as the `raw` column is nullable.
```suggestion
                let raw_tx: Option<Vec<u8>> = row.get("raw")?;
```

Reply:
And indeed this was reverted in a later commit as broken. I cannot tell from the current commits why this was initially considered necessary, and why it is no longer needed.